recipes/stlab/old/conanfile.py

```diff
@@ -0,0 +1,187 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, collect_libs, copy, get, rename, replace_in_file, rmdir, save
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.52.0"
+
+class Stlab(ConanFile):
+    name = 'stlab'
+    description = 'The Software Technology Lab libraries.'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/stlab/libraries'
+    license = 'BSL-1.0'
+    topics = 'conan', 'c++', 'concurrency', 'futures', 'channels'
+
+    settings = "arch", "os", "compiler", "build_type",
+
+    options = {
+        "boost_optional": [True, False],
+        "boost_variant": [True, False],
+        "coroutines": [True, False],
+        "task_system": ["portable", "libdispatch", "emscripten", "pnacl", "windows", "auto"],
+    }
+
+    default_options = {
+        "boost_optional": False,
+        "boost_variant": False,
+        "coroutines": False,
+        "task_system": "auto",
+    }
+
+    no_copy_source = True
+    _source_subfolder = 'source_subfolder'
+
+    # short_paths = True
+    # def layout(self):
+    #     cmake_layout(self, src_folder="src")
+
```

Comment:
IS this is header only it should be using the basic layout :)

Reply:
How?

Reply:
https://github.com/conan-io/conan-center-index/blob/81ab08383840bc9b8a8bb3afac956bc02894f74d/docs/package_templates/header_only/all/conanfile.py#L49